### PR TITLE
add `msglm`

### PR DESCRIPTION
**Pull request by comhar**
This PR integrates the [msglm](https://github.com/AnswerDotAI/msglm/) package.

### CODE REVIEW

File: settings.ini [hunk 1]
```diff
@@ -5,7 +5,7 @@ version = 0.0.10
 min_python = 3.8
 license = apache2
 black_formatting = False
-requirements = fastcore>=1.5.33 anthropic toolslm
+requirements = fastcore>=1.5.33 anthropic toolslm git+ssh://git@github.com/AnswerDotAI/msglm.git
```
Comment (comhar) [inline settings.ini:8]: this will be replaced once `msglm` is available on pypi.

**Comment by jph00** [comment]
`msglm` is released now @comhar so you can update settings.ini.

**Comment by jph00** [comment]
@comhar what's the status of this?

**Comment by jph00** [comment]
@comhar just bumping this

**Comment by comhar** [comment]
closing this PR as it was created before we added the gitnotebooks integration. I'll open a new one now so that we can use gitnotebooks during the review.

**Pull request closed without merging**